parsing/dialects/intermediate/dialects/dialect1/parserImpls.ftl

```diff
@@ -4496,11 +4496,26 @@ SqlNode CreateProcedureStmt() :
 }
 {
     (
+        e = SqlBeginEndCall()
+    |
         e = SqlStmt()
     )
     { return e; }
 }
 
+void CreateProcedureStmtList(SqlStatementList statements) :
+{
+    SqlNode e;
+}
+{
+    (
+        LOOKAHEAD(1)
```

Comment:
Why is LOOKAHEAD(1) required here?

Reply:
When it was using the default lookahead of 2, it was throwing some errors saying it expected tokens like "INSERT ;" when I wouldn't expect it to lookahead for a semicolon. Changing it to LOOKAHEAD(CreateProcedureStmt()) also fixes it so it could be changed to that to make it more explicit?

Reply:
Ok, thanks for the explanation. Let's go with the LOOKAHEAD(CreateProcedureStmt()) option, since it's a bit more explicit.

Reply:
Yeah that sounds good, will update